docs/contributing/featureflags.md

```diff
@@ -86,16 +86,24 @@ feature flag. The team should agree on the scope of what is flagged and where th
 applied - both client-side and server-side. While there is no precise rule on what constitutes a
 "feature", work together on the best balance of flags and their respective purposes.
 
+Once you have decided that a feature flag is necessary, the first step is to decide on a name.
+Recommendations for naming are:
```

Comment:
My concern is that we're asking someone to use `flags.json` as a starting point, and then asking them to put the flag name in the server code.  Throughout that process they wouldn't have any LD interaction, or any indication of how LD prefers to structure the key name.  Then, when moving into LD proper, the developer would have to change the server code (and client as well, potentially) with the new name that LD suggests.  For example, going from `show_feature` or `showFeature` to `show-feature`.  I was trying to avoid that by adding best practices at this step.

I do think it makes sense to pull back in the naming guidance that you had as well, but that is my thinking for being a bit more prescriptive.

Reply:
It also looks like a lot of those bullets referenced the idea that we were going to set up the flags directly in Jira, whereas now the instructions are for someone to reach out to their EM for setup directly in LD.  Do we still need to explicitly tie it to a Jira issue somehow?